decidim-elections/app/controllers/decidim/elections/admin/elections_controller.rb

```diff
@@ -35,7 +35,7 @@ def create
 
             on(:invalid) do
               flash.now[:alert] = I18n.t("elections.create.invalid", scope: "decidim.elections.admin")
-              render action: "new", status: :unprocessable_entity
+              render action: "new"
```

Comment:
I would still want to have the status sent by controller. This would help in the upcoming hotwire integration.

Reply:
ok